packages/flutter/test/painting/image_cache_test.dart

```diff
@@ -458,115 +465,115 @@ void main() {
     final TestImageStreamCompleter completer1 = TestImageStreamCompleter()
       ..addListener(listener);
 
-    imageCache.putIfAbsent(testImage, () => completer1);
-    expect(imageCache.statusForKey(testImage).pending, true);
-    expect(imageCache.statusForKey(testImage).live, true);
-    expect(imageCache.statusForKey(testImage).keepAlive, false);
-    expect(imageCache.currentSizeBytes, 0);
+    imageCache!.putIfAbsent(testImage, () => completer1);
```

Comment:
Not for this PR, but all of this is making me think we should just be making the `imageCache` late non-nullable somehow.